pkg/schema/tracker.go

```diff
@@ -40,17 +40,14 @@ import (
 const (
 	// TiDBClusteredIndex is the variable name for clustered index.
 	TiDBClusteredIndex = "tidb_enable_clustered_index"
-	// TiDBChangeColumnType is the variable name for column type changing.
-	TiDBChangeColumnType = "tidb_enable_change_column_type"
 )
 
 var (
 	// don't read clustered index variable from downstream because it may changed during syncing
 	// we always using OFF tidb_enable_clustered_index unless user set it in config.
 	downstreamVars    = []string{"sql_mode", "tidb_skip_utf8_check"}
 	defaultGlobalVars = map[string]string{
-		TiDBClusteredIndex:   "OFF",
-		TiDBChangeColumnType: "ON", // NOTE The default value of tidb_enable_change_column_type was changed to ON after the release of TiDB 5.1.
```

Comment:
do you notice if we didn't remove this variable, what problem will happen?

Reply:
will painc https://github.com/pingcap/dm/pull/1910#pullrequestreview-719562064
I'm not sure I've missed anything else

Reply:
user may set the variable in config file, and panic is not friendly. you can take a look at https://github.com/pingcap/dm/pull/1962/files or find another way to handle it. Also add tests for it.

please try to keep PR fix one thing, that's easier to review.

Reply:
fixed in https://github.com/pingcap/dm/pull/1910/commits/8035d6ca67b3e6ea57d21d9774ec62598f04ad99